araproc/analysis/standard_reco.py

```diff
@@ -803,16 +801,13 @@ def min_frac_corr_depth(self, corr_map, fraction=0.6, z_thresh=0):
                     depth = radius * math.sin(math.radians(theta)) + avg_z
                     
 
-                    # Ensure min_depth does not exceed z_thresh
-                    if depth > min_depth and depth <= z_thresh:
+                    # Update min_depth 
+                    if depth > min_depth:
```

Comment:
This goes the right way? In case depth is a negative number?

Reply:
Right. I remember this is sweeping downward from the surface (really from z_threshold). Maybe we need to sweep up for negative depths? 

I can do that today if you want

Reply:
I don't have strong opinions about which way you go! I'm really asking about the sign convention I guess.

Reply:
This should be right I think. A more negative depth is closer to Earth's center and you want the shallowest (i.e. most positive) depth where the conditions are satisfied.

Reply:
Okay, then sounds good